tensorboard/webapp/notification_center/_views/notification_center_component.ng.html

```diff
@@ -39,7 +39,7 @@
         </ng-container>
         <span *ngIf="notification.icon">・</span>
         <span [title]="notification.dateInMs | date:'full'"
-          >{{ notification.dateInMs | date:'medium'}}</span
+          >{{ notification.dateInMs | date:'EEE MMM d, y'}}</span
```

Comment:
So I kind of liked the more compact form since `fullDate` can get pretty long and I wanted to ensure there wouldn't be any wrapping:

`Wednesday, September 30, 2021` vs `Wed Sep 30, 2021`

For i18n, the docs say you have to specifically import locale data to get anything that isn't en-US: https://angular.io/api/common/DatePipe#description  To my knowledge we don't do that today - do we?  And it seems a bit unlikely IMO that we're going to start localizing everything in TB; that would be a pretty significant effort in its own right.

But if you feel strongly I can use `mediumDate` and just skip the day of the week field.

Reply:
> To my knowledge we don't do that today - do we? And it seems a bit unlikely IMO that we're going to start localizing everything in TB; that would be a pretty significant effort in its own right.

Our stated stance is, so it is not a huge amount of work when we want it, we want to make our app as i10n friendly as possible where convenient. We have been using proper intl format for some strings for pluralization and dates thus far. If you think `fullDate` is very bad for the UX and cannot be compromised on, I will support the hard coded date format.

Reply:
I'll do `fullDate` for now, but FWIW `Wednesday, September 30, 2021` is long enough that if the category name (What's New) were a little longer, it would wrap.  So if we expand the category names, IMO we should move to `mediumDate`.  I still think it's a bit of a loss to lose the day of the week in that case, but I don't care strongly enough.